cmd/broker/main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 		logger.Fatal("main", err)
 	}
 
-	broker := broker.BlockheadBroker{}
+	broker := &broker.BlockheadBroker{}
```

Comment:
why is this an issue? we only create that object once.

Reply:
That is true that we only use it once. I didn't think about that. I do prefer the pointer receiver but  don't feel strongly about this either way.

Reply:
I would say lets keep it as is. i dont see any added value with the pointer receiver. otherwise good to merge.